app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,48 @@
  *= require foundation_and_overrides
  */
 
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
  /************** BASIC STYLING *****************************/
 
 #content {
-  background-color: #DADADA;
+  background-color: #FFF;
+  margin-bottom: 6rem;
+  margin-top: 6rem;
+}
+
+#background {
+  display: flex;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, display, flex-flow, height, overflow